Member-Travel-Fund.md

```diff
@@ -0,0 +1,40 @@
+## Purpose
+
+To establish and administer a fund for members of the Node.js
+Foundation to travel and spread knowledge about the foundation
+and the Node.js Foundation projects.
+
+## Restrictions
+
+* Candidates must be Individual Members of the Node.js Foundation.
+* Candidates must be presenting one of the following presentations:
+ * [The Node.js Foundation](https://github.com/nodejs/foundation-slides).
+
+## Process
+
+New slide decks are added to the list of approved decks on an ongoing basis.
+
+* Slide deck must be managed by the Node.js Evangelism WG.
+* Send a pull request to this file adding the slides to
+ the list above.
+* Slide deck must be approved by the TSC.
+
+Any member can apply to give an approved talk. Request may take up to a month
+for the TSC to approve or decline.
+
+* Log an issue in this repository.
+ * Include the name of the event you plan to attend.
+ * Include the location of the event and where you will be traveling from.
+ * Include the presentation you intend to give.
+ * Include the size of the stipend you wish to receive.
+  * Stipends are anywhere from $100 to $1,500 USD and vary with travel distance.
+
+The TSC will consider each proposal in its next regular meeting. While it is
+ultimately at the TSC's discretion who to approve the following considerations
+are made.
+
+* Impact: Preference given to underserved communities.
+* Cost: The larger the stipend the more critically the TSC will consider the application.
```

Comment:
If proof of purchase is required then are y'all assuming that it will only be -after- the speaker has attended that this stipend would be rewarded? This is a barrier for some--they can't afford to commit to speaking if they don't know whether it will be covered. I see this quite often in underrepresented groups speakers as they are also more likely to be paid less for their day work. I've experienced many a speaker having a tough time even waiting until the conference date in order to be reimbursed. I understand that can make things much more complicated accounting-wise, however.